materialsLibrary/src/simple/simple.vertex.fx

```diff
@@ -42,6 +42,8 @@ varying vec3 vNormalW;
 
 #ifdef VERTEXCOLOR

```

Comment:
can we use #if defined(VERTEXCOLOR) || defined(INSTANCESCOLOR) to avoid code duplication ?